okhttp-clients/src/main/java/com/palantir/remoting3/okhttp/InstrumentedInterceptor.java

```diff
@@ -16,25 +16,20 @@
 
 package com.palantir.remoting3.okhttp;
 
-import com.codahale.metrics.MetricRegistry;
-import com.codahale.metrics.SharedMetricRegistries;
 import com.palantir.remoting3.okhttp.metrics.HostMetricsRegistry;
+import com.palantir.tritium.metrics.TaggedMetricRegistry;
 import java.io.IOException;
 import okhttp3.Interceptor;
 import okhttp3.Response;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 /**
  * Records metrics about the response codes of http requests.
  */
 public final class InstrumentedInterceptor implements Interceptor {
 
-    private static final Logger log = LoggerFactory.getLogger(InstrumentedInterceptor.class);
-
     private final HostMetricsRegistry hostMetrics;
 
-    InstrumentedInterceptor(MetricRegistry registry, String serviceName) {
+    InstrumentedInterceptor(TaggedMetricRegistry registry, String serviceName) {
```

Comment:
TaggedMetricRegistry should really be an interface and part of the tritium-api so that we can avoid transitive dependencies on all the tritium implementation details. http-remoting already being a shared library, those dependencies make me a little uneasy. in theory, after this merges, tritium can no longer change any of its public APIs.

Reply:
@schlosna , what would it take to make that change?

Reply:
unless we plumb the registry all the way through the remoting apis we're still going to have the issue. explicitly the issue is that we use `TaggedMetricRegistry.getDefault()` to get the global registry at the moment.

Reply:
Can do the serviceloader thing to pull the implementation if we need to.

Reply:
ugh